tests/backfill/test_dir_multiple/backfill.yaml

```diff
@@ -0,0 +1,19 @@
+2023-05-03:
+  start_date: 2023-01-03
+  end_date: 2023-05-03
+  excluded_dates:
+  - 2023-02-03
+  reason: Please provide a reason for the backfill and links to any related bugzilla or jira tickets
+  watchers:
+  - nobody@mozilla.com
+  status: Drafting
+
+2021-05-03:
```

Comment:
What happens if we run a backfill on a specific data and then realize we made a mistake and have to run it again on the same day?

Reply:
The backfill with the specified date could be updated or not if it is a re run.